main/musl-mallocng/template.py

```diff
@@ -24,14 +24,20 @@
 license = "MIT"
 url = "http://www.musl-libc.org"
 source = f"https://git.musl-libc.org/cgit/musl/snapshot/musl-{_commit}.tar.gz"
-sha256 = "5829457efb2247c1e39920b14721b75e9c488a06149736c8317536ec4aa3764b"
+sha256 = "1d75d71482d339ef13cdf7f8ae16e4e818170306ddc5e662f9f72aa71655743f"
 compression = "deflate"
 # scp makes it segfault
 hardening = ["!scp"]
 # does not ship tests
 options = ["!check", "!lto"]
 
 
+def post_extract(self):
+    # reported in libc.so --version
+    with open(self.cwd / "VERSION", "w") as f:
+        f.write(pkgver)
```

Comment:
don't care too much about this (and left it out of cross), just a slightly nicer version display. could put the commit in it instead too..

Reply:
this is ok